taxi/urls.py

```diff
@@ -20,7 +21,19 @@
     path("cars/<int:pk>/", CarDetailView.as_view(), name="car-detail"),
     path("drivers/", DriverListView.as_view(), name="driver-list"),
     path(
-        "drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail"
+        "drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail",
+    ),
+    path(
+        "login/",
+        auth_views.LoginView.as_view(template_name="registration/login.html"),
+        name="login"
+    ),
+    path(
+        "logout/",
+        auth_views.LogoutView.as_view(
+            template_name="registration/logout.html"
```

Comment:
The `LogoutView` does not require a `template_name` because it doesn't render a template. It simply logs the user out and redirects them. Consider removing the `template_name` argument.